core/consensus/sumeragi.cpp

```diff
@@ -193,7 +193,7 @@ namespace sumeragi {
             this->numValidatingPeers = this->validatingPeers.size();
             // maxFaulty = Default to approx. 1/3 of the network.
             this->maxFaulty = config::IrohaConfigManager::getInstance()
-                    .getMaxFaultyPeers(this->numValidatingPeers / 3);
+                    .getMaxFaultyPeers( std::max( uint64_t(0), this->numValidatingPeers / 3 ));
```

Comment:
This is that if numValidatingPeers less than 0, it resolve this problem.
but, this will rewrite to "PeerServiceConfig::getMaxFaulty()".